beta/project-griffon/using-project-griffon/validation-summary-and-project-griffon.md

```diff
@@ -0,0 +1,69 @@
+# Validation Summary
+
+## Overview
+
+You may use the Validation Summary feature to quickly and easily manage JavaScript functions to validate events in a Project Griffon session.
```

Comment:
I would add a little more here about how those functions will search all the data in the session to see if the conditions are met or not.